charts/kube-prometheus-stack/templates/prometheus/serviceThanosSidecar.yaml

```diff
@@ -16,7 +16,9 @@ metadata:
 {{- end }}
 spec:
   type: {{ .Values.prometheus.thanosService.type }}
-  clusterIP: None
+{{- if .Values.prometheus.thanosService.clusterIP }}
```

Comment:
as "None" is set the if is always true and could be removed.

Reply:
Fixed!